esp/src/src/nls/hu/hpcc.ts

```diff
@@ -295,6 +297,7 @@ export = {
     FirstN: "Az első N",
     FirstName: "Keresztnév",
     FirstNRows: "Első N rekord",
+    FirstNSortBy: "Szerinti rendezés",
```

Comment:
I was wrong. The correct one is "Az első N rekord rendezése"

Reply:
Are you sure? The English version is "Sort by"

Reply:
@AttilaVamos yes, @richardkchapman  is correct, it is "Sort By".

Reply:
I see. 
So, if the context is  "Sort by" then the correct one is "Szerinti rendezés"
If the context is "First N sort by" then the "Az első N rekord rendezése" is the right one.